src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "item.mythicmetals.carmot_helmet": "Carmothelm",
+  "item.mythicmetals.carmot_hoe": "Carmothacke",
+  "item.mythicmetals.carmot_ingot": "Carmotbarren",
+  "item.mythicmetals.carmot_leggings": "Carmothose",
+  "item.mythicmetals.carmot_nugget": "Carmotnugget",
+  "item.mythicmetals.carmot_pickaxe": "Carmotspitzhacke",
+  "item.mythicmetals.carmot_shovel": "Carmotschaufel",
+  "item.mythicmetals.carmot_staff": "Carmotstaab",
+  "item.mythicmetals.carmot_sword": "Carmotschwert",
+  "item.mythicmetals.celestium_anvil": "Celestiumamboss",
+  "item.mythicmetals.celestium_axe": "Celestiumaxt",
+  "item.mythicmetals.celestium_boots": "Celestiumstiefel",
+  "item.mythicmetals.celestium_chestplate": "Celestiumbrustplatte",
+  "item.mythicmetals.celestium_dust": "Celestiumstaub",
+  "item.mythicmetals.celestium_elytra": "Celestiumelytren",
+  "item.mythicmetals.celestium_helmet": "Celestiumhelm",
+  "item.mythicmetals.celestium_hoe": "Celestiumhacke",
+  "item.mythicmetals.celestium_ingot": "Celestiumbarren",
+  "item.mythicmetals.celestium_leggings": "Celestiumhose",
+  "item.mythicmetals.celestium_nugget": "Celestiumnugget",
+  "item.mythicmetals.celestium_pickaxe": "Celestiumspitzhacke",
+  "item.mythicmetals.celestium_shovel": "Celestiumschaufel",
+  "item.mythicmetals.celestium_sword": "Celestiumschwert",
+  "item.mythicmetals.copper_axe": "Kupferaxt",
+  "item.mythicmetals.copper_boots": "Kupferstiefel",
+  "item.mythicmetals.copper_chestplate": "Kupferbrustplatte",
+  "item.mythicmetals.copper_dust": "Kupferstaub",
+  "item.mythicmetals.copper_helmet": "Kupferhelm",
+  "item.mythicmetals.copper_hoe": "Kupferhacke",
+  "item.mythicmetals.copper_leggings": "Kupferhose",
+  "item.mythicmetals.copper_nugget": "Kupfernugget",
+  "item.mythicmetals.copper_pickaxe": "Kupferspitzhacke",
+  "item.mythicmetals.copper_shovel": "Kupferschaufel",
+  "item.mythicmetals.copper_sword": "Kupferschwert",
+  "item.mythicmetals.doge": "Doge",
+  "item.mythicmetals.doge.desc": "Was macht Doge?",
+  "item.mythicmetals.durasteel_anvil": "Durastahlamboss",
+  "item.mythicmetals.durasteel_axe": "Durastahlaxt",
+  "item.mythicmetals.durasteel_boots": "Durastahlstiefel",
+  "item.mythicmetals.durasteel_chestplate": "Durastahlbrustplatte",
+  "item.mythicmetals.durasteel_dust": "Durastahlstaub",
+  "item.mythicmetals.durasteel_helmet": "Durastahlhelm",
+  "item.mythicmetals.durasteel_hoe": "Durastahlhacke",
+  "item.mythicmetals.durasteel_ingot": "Durastahlbarren",
+  "item.mythicmetals.durasteel_leggings": "Durastahlhose",
+  "item.mythicmetals.durasteel_engine": "Durastahlmotor",
+  "item.mythicmetals.durasteel_nugget": "Durastahlnugget",
+  "item.mythicmetals.durasteel_pickaxe": "Durastahlspitzhacke",
+  "item.mythicmetals.durasteel_shovel": "Durastahlschaufel",
+  "item.mythicmetals.durasteel_sword": "Durastahlschwert",
+  "item.mythicmetals.enchanted_midas_gold_block": "Verzauberter Midasgoldblock",
+  "item.mythicmetals.froge": "Frobsch",
```

Comment:
Requesting review from @gliscowo regarding if this is a fitting translation for Froge ![Froge](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.redd.it%2Fniflrvzbxpb41.jpg&f=1&nofb=1&ipt=cb087fd1598996420369d252e5e42350255ad683c44b1f282fbe7b1793c8a6bf&ipo=images)

Reply:
Translators note:
"Translating it as „Frobsch“ implies a fundamental misunderstanding of froge and its history. „Frobsch“ seems to come from the misconception that froge is simply frog but with an „e“. We all know however that this could not be further from the truth, for froge is nothing less than the divine amalgamation of a frog and doge. Personally I think it’s name carries enough might to shatter any notion of translation but should one be required I propose a name more conscious of its origin. The frog part is easy since it is just „Frosch“ but since doge is an intentional misspelling of dog, unique issues arise. Misspelling „Hund“ the same way the original does by adding an „e“ is not viable since „Hunde“ is just the plural of „Hund“. I therefore propose „Hünd“ as an affectionate misspelling, with the divine creature itself taking the name „Fründ“ which has the added benefit of sounding like friend or „Freund“, which befits its charming and friendly character."

Reply:
Bumping this review comment, as it is fairly easy to change